apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/TextResponseBlockNode/TextResponseBlock.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import { MockedProvider } from '@apollo/client/testing'
+import { render, screen } from '@testing-library/react'
+import { Position, ReactFlowProvider } from 'reactflow'
+
+import { TreeBlock } from '@core/journeys/ui/block'
+
+import {
+  BlockFields_StepBlock as StepBlock,
+  BlockFields_TextResponseBlock as TextResponseBlock
+} from '../../../../../../../__generated__/BlockFields'
+import {
+  ThemeMode,
+  ThemeName
+} from '../../../../../../../__generated__/globalTypes'
+import { mockReactFlow } from '../../../../../../../test/mockReactFlow'
+import '../../../../../../../test/i18n'
+
+import { TextResponseBlockNode } from './TextResponseBlockNode'
+
+describe('TextResponseBlock', () => {
+  beforeEach(() => {
+    mockReactFlow()
+  })
+
+  const mockTextResponseBlock = {
+    __typename: 'TextResponseBlock',
+    id: 'TextResponseBlock.id',
+    parentBlockId: 'CardBlock.id',
+    parentOrder: 0,
+    label: 'TextResponseBlock',
+    hint: null,
+    minRows: null,
+    submitLabel: null,
+    submitIconId: null,
+    action: null,
+    children: []
+  } satisfies TreeBlock<TextResponseBlock>
```

Comment:
Wow, didn't know `satisfies` was a thing! I'm still looking into exactly how it works, but I dont think its necessary here.

```suggestion
  const mockTextResponseBlock: TreeBlock<TextResponseBlock> = {
    __typename: 'TextResponseBlock',
    id: 'TextResponseBlock.id',
    parentBlockId: 'CardBlock.id',
    parentOrder: 0,
    label: 'TextResponseBlock',
    hint: null,
    minRows: null,
    submitLabel: null,
    submitIconId: null,
    action: null,
    children: []
  }
```

Reply:
Yeah, it's a pretty neat type! I'm fine switching to the annotation since it's really the same in this instance. The main idea is to provide 'narrow' types vs 'wider' types which is pretty nice. https://www.learningtypescript.com/articles/the-satisfies-operator this article does a great job of explaining it.